src/components/icons/settings.js

```diff
@@ -12,6 +12,9 @@ const SettingsIcon = ({ className }) => (
       <path d="M2.8 2.05L4.3 2.05 4.55 5.05 2.55 5.05z" />
       <path d="M12.7 11.95L14.2 11.95 14.45 14.95 12.45 14.95z" />
       <path d="M15.5 8.389c0 3.866-3.134 7-7 7s-7-3.134-7-7 3.134-7 7-7 7 3.134 7 7zm-2 0c0-2.762-2.239-5-5-5s-5 2.238-5 5c0 2.761 2.239 5 5 5s5-2.239 5-5z" />
+      {hasReminder && (
+        <circle id="Oval" fill="#FFBC00" cx="16" cy="4" r="4"></circle>
```

Comment:
I wonder if we should use the `$color-accent-yellow` variable here instead of hardcoding it?

Reply:
I hardcoded this as there are no dynamic assignment of this color hence there isn't any need to adopt a variable here.

Setting the color variable entails setting up another css class as well as the need to set up another variable to pass within the `icon-button` component, which I am trying to avoid.